blueocean-commons/src/main/java/io/jenkins/blueocean/commons/ResourcesUtils.java

```diff
@@ -0,0 +1,46 @@
+package io.jenkins.blueocean.commons;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Collectors;
+
+public class ResourcesUtils {
+    public ResourcesUtils() {
+        // no op
+    }
+
+    /**
+     * use {@link #toString(URL, Charset)} using the default charset {@link StandardCharsets#UTF_8}
+     * @see #toString(URL, Charset)
+     */
+    public static String toString(URL resource)
+        throws IOException
+    {
+        return toString(resource, StandardCharsets.UTF_8);
+    }
+
+    /**
+     *
+     * @param resource the URL resource
+     * @param charset the charset to use
+     * @return turn the  URL resource into a String using the given Charset
+     * @throws IOException if any issue reading the resource
+     */
+    public static String toString(URL resource, Charset charset)
+        throws IOException
+    {
+        try (InputStream inputStream = resource.openStream();
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader( inputStream, charset))){
+            return bufferedReader
+                .lines()
+                .collect( Collectors.joining( "\n"));
```

Comment:
this looks funky, you are loading a file, parsing it for line ends, splitting it, then joining it all back again?
Why not use `apache commons IOUtils.toString(URI uri, Charset encoding)` or other equivalent helpers and avoid the intermediate parsing/splitting?

Reply:
I love funky :) if not lines are collected in a single line 
Thanks I didn't know this IOUtils method and will use now